src/app/auth.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Subject } from 'rxjs';
+
+export class AuthService {
+	public loggedIn = new Subject<boolean>();
+	public filteredStatus = new Subject<string>();
+
+	checkLoggedInStatus() {
+		return this.loggedIn.asObservable();
+	}
+
+	setLoginStatus(status: boolean) {
+		this.loggedIn.next(status);
+	}
+
+	returnFilteredStatus() {
```

Comment:
use es feature like 
returnFilteredStatus = () => this.filteredStatus.asObservable();